azure-pipelines/build.yml

```diff
@@ -0,0 +1,127 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+name: $(Date:yyyyMMdd).$(Rev:r)
+variables:
+- name: ApiScanClientId
+  value: 8cce479d-aee7-48ad-a21a-547253d6f599
+- name: ApiScanEnabled
+  value: true
+- name: ApiScanSecret
+  value: ''
+- name: ApiScanTenant
+  value: 72f988bf-86f1-41af-91ab-2d7cd011db47
+- name: AzureServicesAuthConnectionString
+  value: ''
```

Comment:
Do we need any of this? Isn't that now handled by the template?

Reply:
Correct, these should get removed.